src/assets/text/droughtAnnotations_desktop.js

```diff
@@ -68,11 +87,36 @@ export default {
         desktop_text_anchor: 'start',
         desktop_text_width: 100
       },
+      {
+        id: 'map',
+        date: '1923-02-01',
+        text: 'Map of streamgage locations that had one of the 2000 most severe droughts.',
+        desktop_x_offset_per: 60,
+        desktop_text_anchor: 'start',
+        desktop_text_width: 200
+      },
+      {
+        id: 'mapHighlight',
+        date: '1930-09-01',
+        text: 'Streamgage locations in drought during the Dust Bowl are highlighted on the map in red.',
```

Comment:
This comes in at the right time but if you keep scrolling is out of place 
<img width="1335" alt="image" src="https://github.com/DOI-USGS/drought-timeline/assets/17803537/9c159326-6d00-40c4-8133-db6e0130f1e0">
It needs to either be sticky to the map or moved elsewhere